### PR TITLE
Made HW8, p. 1-2

### PR DESCRIPTION
**Pull request by anastasiya-harbatsevich**
p1. Bedroom
p2. Printer

### CODE REVIEW

File: src/main/java/org/example/homework8/Bedroom/Bed.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package org.example.homework8.Bedroom;
+
+public class Bed {
+    private String style;
+    private int pillows;
+    private int height;
+    private int sheets;
+    private int quilt;
+    public Bed(String style, int pillows, int height, int sheets, int quilt) {
```
Comment (alexshamrai) [inline src/main/java/org/example/homework8/Bedroom/Bed.java:10]: Split constructor with empty line, pls

File: src/main/java/org/example/homework8/Bedroom/Bedroom.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        this.wall3 = wall3;
+        this.wall4 = wall4;
+        this.ceiling = ceiling;
+        this.bed = bed;
+        this.lamp = lamp;
+    }
+
+    public static void makeBed() {
+        Bed.make();
+//        System.out.println("The bed is being made");
```
Comment (alexshamrai) [inline src/main/java/org/example/homework8/Bedroom/Bedroom.java]: Why comment? The idea of the task was to leave something like a log message. e.g. `"Bedrom -> start making the bed"`, in order to trace which class and method are executed

File: src/main/java/org/example/homework8/Bedroom/Bed.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        this.sheets = sheets;
+        this.quilt = quilt;
+    }
+
+    public static void make(){
+        System.out.println("The bed is being made");
+    }
+
+    public String getStyle(){
+        return this.style;
```
Comment (alexshamrai) [inline src/main/java/org/example/homework8/Bedroom/Bed.java]:
```suggestion
        return style;
```
using this in getters is not mandatory. i guess it looks better without it here

File: src/main/java/org/example/homework8/Bedroom/Ceiling.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+public class Ceiling {
+    int height;
+    int paintedColour;
+    public Ceiling(int height, int paintedColour) {
+        this.height = height;
+        this.paintedColour = paintedColour;
+    }
+
+    public int getHeight(){
+        return this.height;
```
Comment (alexshamrai) [inline src/main/java/org/example/homework8/Bedroom/Ceiling.java]: the same about this in getters, here and below

**Review by alexshamrai — CHANGES_REQUESTED** [review]
small comments, but better to fix

File: src/main/java/org/example/homework8/Bedroom/Bed.java (hunk 1 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework8/Bedroom/Bed.java:10]: Lines are splitted.

File: src/main/java/org/example/homework8/Bedroom/Bedroom.java (hunk 2 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework8/Bedroom/Bedroom.java]: Fixed.

File: src/main/java/org/example/homework8/Bedroom/Bed.java (hunk 3 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework8/Bedroom/Bed.java]: Improved.

File: src/main/java/org/example/homework8/Bedroom/Ceiling.java (hunk 4 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework8/Bedroom/Ceiling.java]: Improved.

File: src/main/java/org/example/homework8/Bedroom/Bed.java (hunk 1 above)
Comment (alexshamrai) [reply src/main/java/org/example/homework8/Bedroom/Bed.java:10]: i mean not this just sepearate constructor from fields with empty line)))
not every field from other field. 
This is what i wanted
```
    private String style;
    private int pillows;
    private int height;
    private int sheets;
    private int quilt;

    public Bed(String style, int pillows, int height, int sheets, int quilt) {
        this.style = style;
        this.pillows = pillows;
        this.height = height;
        this.sheets = sheets;
        this.quilt = quilt;
    }
```

File: src/main/java/org/example/homework8/Bedroom/Bedroom.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
+package org.example.homework8.Bedroom;
+
+public class Bedroom {
```
Comment (alexshamrai) [inline src/main/java/org/example/homework8/Bedroom/Bedroom.java:3]:
```
public class Bedroom {
    String name;
    Wall wall1;
    Wall wall2;
    Wall wall3;
    Wall wall4;
    Ceiling ceiling;
    Bed bed;
    Lamp lamp;

    public Bedroom(String name, Wall wall1, Wall wall2, Wall wall3, Wall wall4, Ceiling ceiling, Bed bed, Lamp lamp) {
        this.name = name;
        this.wall1 = wall1;
        this.wall2 = wall2;
        this.wall3 = wall3;
        this.wall4 = wall4;
        this.ceiling = ceiling;
        this.bed = bed;
        this.lamp = lamp;
    }
```

File: src/main/java/org/example/homework8/Bedroom/Ceiling.java [hunk 6]
```diff
@@ -0,0 +1,17 @@
+package org.example.homework8.Bedroom;
+
+public class Ceiling {
+    int height;
+    int paintedColour;
```
Comment (alexshamrai) [inline src/main/java/org/example/homework8/Bedroom/Ceiling.java:5]: Now you see?
```
    int height;
    int paintedColour;

    public Ceiling(int height, int paintedColour) {
        this.height = height;
        this.paintedColour = paintedColour;
    }
```

File: src/main/java/org/example/homework8/Bedroom/Lamp.java [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+public class Lamp {
+
+    private String style;
+    private boolean battery;
+    private int globRating;
+
+    public Lamp(String style, boolean battery, int globRating) {
+        this.style = style;
+
```
Comment (alexshamrai) [inline src/main/java/org/example/homework8/Bedroom/Lamp.java]: remove this empty line

File: src/main/java/org/example/homework8/Bedroom/Lamp.java [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+        this.battery = battery;
+
```
Comment (alexshamrai) [inline src/main/java/org/example/homework8/Bedroom/Lamp.java]: remove empty line

**Review by alexshamrai — APPROVED** [review]

File: src/main/java/org/example/homework8/Bedroom/Bed.java (hunk 1 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework8/Bedroom/Bed.java:10]: Fixed.

**Pull request merged**